### PR TITLE
Add Separator component

### PR DESCRIPTION
**Pull request by kaevupoiss**
I have tried to implement the separator component. It doesn't seem like I have to return anything else from the `createSeparator()` builder.

### CODE REVIEW

File: src/lib/builders/separator/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { writable } from "svelte/store";
+
+type SeparatorArgs = {
+    orientation?: 'horizontal' | 'vertical';
+    decorative?: boolean;
+}
+
+export function createSeparator(args: SeparatorArgs = {}) {
+    const orientation = writable(args.orientation ?? 'horizontal');
+    const decorative = writable(args.decorative ?? 'false');
```
Comment (TGlide) [inline src/lib/builders/separator/index.ts:11]: These should be returned so they can be modified after creation

File: src/routes/docs/builders/separator/preview.svelte [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { Docs } from '$routes/(components)';
+	import { createSeparator } from '@melt-ui/svelte/builders/separator';
```
Comment (TGlide) [inline src/routes/docs/builders/separator/preview.svelte:3]: Should be $lib. Also, don't forget to export it from '$lib/builders/index.ts' as well

**Review by TGlide — CHANGES_REQUESTED** [review]
Thank you for the PR! Just some small changes needed 😄

**Comment by huntabyte** [comment]
Closing this in favor of #88 - thank you!

**Pull request closed without merging**